CPAN/buildme.sh

```diff
@@ -390,9 +417,9 @@ function build_module {
         
         $PERL_BIN Makefile.PL INSTALL_BASE=$PERL_BASE $makefile_args
         if [ $local_run_tests -eq 1 ]; then
-            make test
+            $GNUMAKE test
```

Comment:
@chincheta0815 - GNUMAKE is only defined for BSD/Soldaris... Could you please fix this?

Reply:
You pushed to the repo "export GNUMALE=$MAKE" as a default right?
So no action item for me anymore?

Reply:
There is: can't GNUMAKE be removed? It's being set to the same value as MAKE anyway?

Reply:
Ah, now I understand...

As it compiles correctly on OmniOS/IllumOS right now I would say yes it could be removed. I can't tell how that works on FreeBSD. I will have a look into that and check I am right for Solaris. I just thought the FreeBSD MAKE/GNUMAKE part to be good idea (see below)

On the "real" Oracle Solaris (it has a different make by default) I remember that I once had to change "some" makes to be gnu makes for compatibility. So maybe it would make sense to leave that or some similar functionality in the code. Could be one MAKE var that is globally gnumake and can be changed in case or so...